### PR TITLE
initial meeting notes

### PR DESCRIPTION
**Pull request by qmx**

### CODE REVIEW

**Comment by eddyb** [comment]
Hmm, I was imagining we'd end up with an issue with checkboxes for action items.

**Comment by qmx** [comment]
@eddyb we definitely can do that - I just want to make sure I've captured things correctly first :)

File: minutes/wg-grammar-meetings/2018.11.14.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	- inline unit tests
+		- We've decided to go with tests in separate files, mimicking the grammar file tree structure, since this brings us the advantage of being able to even had AST/parse tree expectation files
+		- The initial proposed test file name scheme is `<Production>.<random_description>.<ext>`
+	- testing against external corpus
+		- The initial testing corpus will be rust-lang/rust, once we get to a good place we'll extend this to maybe top 100 crates.io (stable)
+3. how to deal with ambiguity (declarative...)
+	- eddyb thinks that's too early to discuss this, we're punting this to the near future
+4. GLL
+	- formatting
+		- we asked for small changes on the overall feel for GLL grammars, [GLL PR](https://github.com/lykenware/gll/pull/74) with the proposed changes.
```
Comment (Centril) [inline minutes/wg-grammar-meetings/2018.11.14.md]: should note what those were:

```rust
FnArgs =
  | Regular:FnArg+ % ","
  | Variadic:"..."
  | RegularAndVariadic:{ args:FnArg+ % "," "," "..." }
  ;
```

i.e. leading `|` and then just moving `|` to the start of line.

File: minutes/wg-grammar-meetings/2018.11.14.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	- syntax changes
+		- there were some discussions about using BNF-like `::=` for attribution of new rules but this went nowhere at least for now.
+
```
Comment (Centril) [inline minutes/wg-grammar-meetings/2018.11.14.md]: We discussed parameterized grammars as well for the future.

**Review by Centril — COMMENTED** [review]
Some minor additions needed :)

**Review by Centril — APPROVED** [review]

**Pull request merged**